cmd/bnscli/cmd_gov.go

```diff
@@ -498,13 +499,20 @@ Creates a new version for an existing election rule. The new version is used for
 	if err := fraction.Validate(); err != nil {
 		flagDie("invalid voting period: %s", err)
 	}
+
+	var quorum *gov.Fraction
+	if f := quorumFl.Fraction(); f.Numerator != 0 {
```

Comment:
Ah... okay, if you set to 0, it ignore this value (not deletes it).
Can you improve the comment?